code/style.css

```diff
@@ -1,7 +1,261 @@
+  height: 50px;
+  margin: 8px;
+  border-radius: 50%;
+  border: 1px solid rgb(235, 100, 100);
+}
+
+.nav-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: black; /* For browsers that does not support gradients*/
+
+}
+
+a {
+  padding: 1rem;
+  font-size: 1.1rem;
+  cursor: pointer;
+  color: rgb(238, 185, 194);
+  text-decoration: none;
+  font-weight: bold;
+}
+
+a:hover {
+  transform: scale(1.2, 1.2);
+}
+
+/* FORM SECTION */
+
+.form-section {
+  margin:20px 10px;
+  padding:40px;
+  background-color: rgb(60, 141, 173);
+  border-radius: 20% 10%;
+}
+
+.register-heading {
+  text-align: center;
+  font-family: 'Merriweather', serif;
+  margin-bottom: 25px;
+  color: rgb(252, 240, 200);
+  text-transform: uppercase;
+}
+
+label {  
+  color: rgb(252, 240, 200);
+  font-weight: bolder;
+
+}
+
+.labels {
+  display:block;
+  padding-left: 10px;
+  margin: 10px 0px;
+}
+
+.inputs {
+  display: block;
+  width: 100%;
+  height: 40px;
+  border:none;
+  margin-bottom: 20px;
+}
+/* Pseudo-class selector: select elements based on a certain state */
+.inputs:focus { 
+   outline: none; /*removes default outline of all inputs */
+}
+/* Pseudo-element selector: select and style a part of an element*/
+input::placeholder { 
+  padding-left: 15px;
+  opacity:0.5;
+}
+/* Attribute selectors: select elements based on an attribute or attribute value */
+input[type="checkbox"] {
+  cursor: pointer;
+}
 
-.active + .answer {
-  /* some styling here */
+.button {
+  display: block;
+  margin:20px auto;
+  width: 90%;
+  height: 50px;
+  background-color: rgb(18, 93, 152);
+  border:none;
+  border-radius: 20px;
+  color: rgb(252, 240, 200);
+  font-size: 24px;
+  cursor: pointer;
+}
+
+.button:hover {
+  transform: scale(1.1, 1.1);
+}
+
+/* ACCORDION SECTION */
+
+.accordion-section {
+  margin:20px 10px;
+  padding:30px;
+  background-color: rgb(145, 31, 39);
+  border-radius: 10% 20%;
+}
+
+.faq-heading {
+  font-family: 'Merriweather', serif;
+  text-align: center;
+  color: rgb(252, 240, 200);
+  margin-bottom: 20px;
+}
+
+.question {
+  width: 100%;
+  margin-top: 10px;
+  background-color: rgb(252, 240, 200);
+  color:rgb(99, 10, 16);
+  cursor: pointer;
+  border-radius: 20px;
+  padding:10px 18px 30px 15px;
+
+}
+
+.question:hover {
+  background-color:rgb(242, 187, 123);
+}
+
+.question::after {
+  content: '+';
+  font-size: 1.4rem;
+  color:rgb(99, 10, 16);
+  float: right;
+}
+
+.active::after {
+  content: '-';
+}
+ 
+.answer { 
+  display: none;
+  color:rgb(250, 206, 127);
+  padding: 5px;
+}
+
+.active + div.answer { /* Adjacent Sibling Selector (+) */
+  display: block;
+}
+
+/* FOOTER */
+.footer {
+  background-color: #000;
+  height: 200px;
+  text-align: center;
+}
+
+.logo-footer {
+  width: 50px;
+  height: 50px;
+  margin-top: 30px;
+  border-radius: 50%;
+  border: 1px solid rgb(235, 100, 100);
+}
+
+.fab { 
+  color:#fff;
+  padding:20px 10px;
+}
+
+
+/* MEDIA QUERIES */
```

Comment:
Your media queries are very good. I get the "hang of it" from looking at your work. 
You keep it simple and that is the key, I guess